### DISCUSSION TITLE
Adaptive Icon for custom Icons that we can select from Appearance - Icons

### DISCUSSION THREAD

**Discussion by pallaviSri2024** [libre-tube/LibreTube#5906] [General]
### Steps to reproduce

Go to Appearance
Select Icons
Choose any custom icon

### Expected behavior

All custom icon should have monochrome icon support as the default LibreTube Icon.

For reference, you can check how Molly implements this feature.

[Link to Molly Project](https://github.com/mollyim/mollyim-android)

### Actual behavior

When the themed icon is enabled, your custom icon will not change into monochrome icon.

### LibreTube version

0.22.2

### Android version

Android 14

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Some of these icons are PNGs. We would first need someone that creates SVG versions of these icons first, and then it would be possible without issues.

**Comment by pallaviSri2024** [comment]
@Bnyro  I can Create SVGs for a few icons, I hope this will resolve this issue.

**Reply by Bnyro** [reply]
Yes, that will help :+1:
